corehq/apps/export/models/new.py

```diff
@@ -1023,6 +1023,7 @@ def get_filters(self):
                 self.filters.accessible_location_ids,
                 self.filters.show_all_data,
                 self.filters.show_project_data,
+                self.filters.show_deactivated_data,
```

Comment:
Same comment as above:

I added this because of the modification I made to the inputs [here](https://github.com/dimagi/commcare-hq/pull/22179/commits/8ad32ae1fa0fffb8bccb18a2c909ce79f4968c4c#diff-628ce8b77792cabb27a638d59a0b064fR765). I tested doing an export with the deactivated filter, I think that tests this code but I'd like a second opinion on that!

Reply:
I don't know of another way that this would be run.